Gemfile

```diff
@@ -14,8 +14,14 @@ if ENV["GEMFILE_MOD"]
   instance_eval(ENV["GEMFILE_MOD"])
 else
   group :development do
-    # temporarily we only support building against master
-    gem "chef", github: "chef/chef", branch: "master"
-    gem "ohai", github: "chef/ohai", branch: "master"
+    # chef 17 is on 3.0
+    # chef 18 is on 3.1
+    if RUBY_VERSION =~ /^3\.1/
```

Comment:
Shouldn't we instead check for 3.0 and use 17; with an 'else' of 18.0 for 3.1 and later?

Reply:
Good call. And I thought about that a little while ago. 🤔